common/containers/Tabs/GenerateWallet/components/Keystore/EnterPassword.scss

```diff
@@ -3,6 +3,10 @@
 $pw-max-width: 40rem;
 
 .EnterPw {
+  input {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+  }
```

Comment:
Can we get a fix for this that works for all input groups, rather than fixing this one instance? e.g.
```
.input-group {
  > * {
    border-radius: 0;

    &:first-child {
      // bottom left / top left have border radius
    }

    &:last-child {
      // top right / bottom right have border radius
    }
  }
}
```

Reply:
This fix was actually specific just to the password inputs when generating a wallet because they're the only inputs with an inline button.

I'll add a proper class to cover this case for any future inputs with inline buttons. Also the border radius for all inputs is 2px.

Before:
<img width="508" alt="screen shot 2018-03-07 at 9 38 16 am" src="https://user-images.githubusercontent.com/13836920/37098315-9b6228ba-21eb-11e8-96c1-ce7e556f0c2e.png">

After:
<img width="372" alt="screen shot 2018-03-07 at 9 39 03 am" src="https://user-images.githubusercontent.com/13836920/37098338-a5155634-21eb-11e8-9495-4e5d5543afc0.png">

Reply:
This style of addon next to input is actually used in a bunch of places, just grep for `input-group-addon`. Mnemonic generation, ENS, and anything that uses the TogglablePassword component. So I think a holistic fix would be ideal.

I was under the impression Bootstrap did this for us (See the `input-group` example here http://getbootstrap.com/docs/3.3/css/#forms-inline), so we may be overriding that somewhere.

Reply:
Nope, since updating the input styles, we've moved on from using the bootstrap styles

Reply:
Oh right on. Well maybe we can carry over what they're doing in that example for border radii.